dashboard/test/ui/features/curriculum_platform/levelbuilder/lesson_edit_page.feature

```diff
@@ -90,6 +90,8 @@ Feature: Using the Lesson Edit Page
     And I wait until element ".progress-bubble" contains text "1"
     Then element ".progress-bubble" contains text "2"
 
+    And I delete the temp unit with lessons
```

Comment:
Thank you for adding these lines to clean up the added scripts! I believe this kind of omission is likely the cause for the growing number of scripts on the test machine. 

for context, we don't have an explicit goal to "load test" the test machine with an ever growing number of scripts, rather it happened inadvertently and we kind of decided that it might be a good idea to let it serve as a load test so that we are not surprised by performance problems the next time we add a bunch of scripts.

anyway, I'm personally on board with fixing this kind of cleanup bug, resetting the test DB, and saving ourselves the trouble of debugging this elusive problem whenever the number of scripts grows too large.

Reply:
> it might be a good idea to let it serve as a load test so that we are not surprised by performance problems the next time we add a bunch of scripts

A thought for the future: if we did want to make sure we're not adding too many scripts, it might be worth seeding all of our scripts on test. We currently only seed a subset of them as seeding takes a long time but it could also cause us to miss performance issues

Reply:
Great idea Bethany! once the current work to move all scripts onto the new seeding process is complete, it may become practical to do this from a performance standpoint. tracking here: [PLAT-1291](https://codedotorg.atlassian.net/browse/PLAT-1291)